modal_proto/api.proto

```diff
@@ -360,6 +360,12 @@ message AppHeartbeatRequest {
   string app_id = 1;
 }
 
+message AppLayout {
+  repeated Object object = 1;
+  map<string, string> function_ids = 2;  // tag -> function id
```

Comment:
I know it's just a comment but should we use different terminology for the keys since we're trying to deconflict "tag" with the deployment tag concept?

Reply:
Yeah makes sense. Any thoughts on the best term?